Sources/SwiftSyntaxMacrosTestSupport/Assertions.swift

```diff
@@ -152,8 +208,10 @@ public struct DiagnosticSpec {
   ///   - highlights: If not empty, the text fragments the diagnostic is expected to highlight
   ///   - notes: The notes that are expected to be attached to the diagnostic
   ///   - fixIts: The messages of the Fix-Its the diagnostic is expected to produce
-  ///   - originatorFile: The file at which this ``NoteSpec`` was created, so that assertion failures can be reported at its location.
+  ///   - originatorFileID: The file ID at which this ``NoteSpec`` was created, so that assertion failures can be reported at its location.
+  ///   - originatorFilePath: The file path at which this ``NoteSpec`` was created, so that assertion failures can be reported at its location.
   ///   - originatorLine: The line at which this ``NoteSpec`` was created, so that assertion failures can be reported at its location.
+  ///   - originatorColumn: The column at which this ``NoteSpec`` was created, so that assertion failures can be reported at its location.
```

Comment:
```suggestion
  ///   - originatorFileID: The file ID at which this ``DiagnosticSpec`` was created, so that assertion failures can be reported at its location.
  ///   - originatorFilePath: The file path at which this ``DiagnosticSpec `` was created, so that assertion failures can be reported at its location.
  ///   - originatorLine: The line at which this ``DiagnosticSpec `` was created, so that assertion failures can be reported at its location.
  ///   - originatorColumn: The column at which this ``DiagnosticSpec `` was created, so that assertion failures can be reported at its location.
```